docker/Dockerfile.chronograf.prod

```diff
@@ -1,5 +1,5 @@
 # from node lts alpine
-FROM node:lts-alpine AS repo
+FROM node:lts-alpine3.14 AS repo
```

Comment:
I may be reading [this release](https://alpinelinux.org/posts/Alpine-3.14.0-released.html) wrong, but it looks like this will put us on Node 14? 

Do we want [`alpine3.15`](https://alpinelinux.org/posts/Alpine-3.15.0-released.html) instead?

Reply:
according to the docker tag itself, `alpine3.15` is on node v18 🤷 
3.15: https://hub.docker.com/layers/library/node/lts-alpine3.15/images/sha256-7b47aea5e936bb7d848fdc29ef3d5afb8835fa074eadcf082680b5f0cfb8f00b?context=explore

3.14: https://hub.docker.com/layers/library/node/lts-alpine3.14/images/sha256-e3c033748d2c218ab9a12d17c6ad9e25756be6517a45f33b175d418fa8b5582d?context=explore